examples/llama-bench/llama-bench.cpp

```diff
@@ -96,6 +96,9 @@ static std::string get_cpu_info() {
                 }
             }
         }
+        if (0 != fclose(f)) {
+            fprintf(stderr, "could not close reader of /proc/cpuinfo\n");
+        }
```

Comment:
We don't check the result of fclose anywhere else, so we shouldn't do it here either. It only has significance if you are writing to a file on a remote filesystem, anyway.

Reply:
I understand that. This PR is a gift and maybe a (good) start.

Reply:
While `/proc/cpuinfo` is not a file that can fail anything ( if it does, it's the least of your worries ) 

An assert on `fclose` in general, wouldn't hurt, as failing `fclose` usually means other problems will follow anyway.

I mean, not in this particular case,  but failing `fclose` often hints full disk drive, or something similar, which can be potentially made worse by continuing execution.

Just a thought.